### PR TITLE
[App Configuration] Update App Configuration tasks description

### PR DESCRIPTION
**Pull request by MaryanneNjeri**
**Task name**: App Configuration Tasks [App Configuration Import task, App Configuration Export task]

**Description**:  Update tasks description and help url

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureAppConfigurationImportV10/task.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-    "helpUrl": "https://learn.microsoft.com/azure/azure-app-configuration/push-kv-devops-pipeline",
+    "description": "Import key-values to an Azure App Configuration instance",
+    "helpUrl": "https://learn.microsoft.com/azure/azure-app-configuration/azure-pipeline-import-task",
     "helpMarkDown": "Email AzureAppConfig@microsoft.com for questions.",
     "category": "Utility",
     "author": "Microsoft Corporation",
     "version": {
         "Major": 10,
-        "Minor": 247,
+        "Minor": 249,
```
Comment (ChristineWanjau) [inline Tasks/AzureAppConfigurationImportV10/task.json:12]: Just curious, how do we decide the minor version?

File: Tasks/AzureAppConfigurationImportV10/task.json [hunk 2]
```diff
@@ -283,7 +283,7 @@
         "InvalidTagsWithSample": "Invalid format for parameter 'Tags'. Sample 'Tags': {\"name1\": \"value1\", \"name2\":, \"value2\"}.",
         "InvalidAppConfigurationEndpoint": "Invalid App Configuration endpoint %s. Please provide a valid App Configuration endpoint.",
         "InvalidTypeInTags": "Invalid type in parameter 'Tags'. Only strings supported",
-        "AzureAppConfigurationPushTaskStartingUp": "Azure App Configuration Push Task is starting up...",
+        "AzureAppConfigurationPushTaskStartingUp": "Azure App Configuration Import Task is starting up...",
```
Comment (ChristineWanjau) [inline Tasks/AzureAppConfigurationImportV10/task.json:286]: The string name should be "AzureAppConfiguration**Import**TaskStartingUp"

**Comment by ChristineWanjau** [comment]
We should also update the user-agent name - https://github.com/microsoft/azure-pipelines-tasks/blob/cb801a211298e4f4aae4127ddd232412a055cbb6/Tasks/AzureAppConfigurationImportV10/utils.ts#L9

**Comment by ChristineWanjau** [comment]
We should update the third party notices text https://github.com/microsoft/azure-pipelines-tasks/blob/cb801a211298e4f4aae4127ddd232412a055cbb6/Tasks/AzureAppConfigurationImportV10/ThirdPartyNotices.txt#L4

File: Tasks/AzureAppConfigurationImportV10/task.json (hunk 1 above)
Comment (MaryanneNjeri) [reply Tasks/AzureAppConfigurationImportV10/task.json:12]: Minor version is based on the current sprint version please see-> https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md

**Comment by MaryanneNjeri** [comment]
> We should also update the user-agent name -
> 
> https://github.com/microsoft/azure-pipelines-tasks/blob/cb801a211298e4f4aae4127ddd232412a055cbb6/Tasks/AzureAppConfigurationImportV10/utils.ts#L9

@ChristineWanjau nice catch this has been updated, thanks

File: Tasks/AzureAppConfigurationImportV10/task.json (hunk 2 above)
Comment (MaryanneNjeri) [reply Tasks/AzureAppConfigurationImportV10/task.json]: True, this has been updated, thanks

**Comment by MaryanneNjeri** [comment]
> We should update the third party notices text
> 
> https://github.com/microsoft/azure-pipelines-tasks/blob/cb801a211298e4f4aae4127ddd232412a055cbb6/Tasks/AzureAppConfigurationImportV10/ThirdPartyNotices.txt#L4

@ChristineWanjau  thanks for catching this has been updated

**Review by max-zaytsev — APPROVED** [review]

**Review by ChristineWanjau — APPROVED** [review]

**Pull request merged**